src/components/header/_header.scss

```diff
@@ -0,0 +1,274 @@
+    margin-right: 1px;
+    fill: currentColor;
+    vertical-align: middle;
+  }
+
+  .govuk-header__logotype-crown-fallback-image {
+    width: 36px;
+    height: 32px;
+    border: 0;
+    vertical-align: middle;
+  }
+
+  .govuk-header__title {
+    @include govuk-font-regular-24;
+  }
+
+  .govuk-header__link {
+    @include govuk-focusable-fill;
+
+    text-decoration: none;
+
+    &:link,
+    &:visited {
+      color: $govuk-header-link;
+    }
+
+    &:hover {
+      text-decoration: underline;
+    }
+
+    &:focus {
+      color: $govuk-black;
+    }
+
+    // alphagov/govuk_template includes a specific a:link:focus selector
+    // designed to make unvisited links a slightly darker blue when focussed, so
+    // we need to override the text colour for that combination of selectors.
+    @include govuk-compatibility(govuk_template) {
+      &:link:focus {
+        @include govuk-text-colour;
+      }
+    }
+  }
+
+  .govuk-header__link--homepage {
+    // Font size needs to be set on the link so that the box sizing is correct
+    // in Firefox
+    @include govuk-typography-common;
+    @include govuk-typography-weight-bold;
+
+    display: inline-block;
+    font-size: 30px; // We don't have a mixin that produces 30px font size
+    line-height: 30px;
+
+    &:link,
+    &:visited {
+      margin-bottom: -1px; // Negate transparent bottom border
+      border-bottom: 1px solid transparent;
+      text-decoration: none;
+    }
+
+    &:hover,
+    &:active {
+      border-bottom-color: currentColor;
+    }
+  }
+
+  .govuk-header__link--service-name {
+    display: inline-block;
+    margin-bottom: $govuk-spacing-scale-2;
+    @include govuk-font-bold-24;
+  }
+
+  .govuk-header__logo {
+    @include govuk-responsive-margin($govuk-spacing-responsive-2, "bottom");
+    padding-right: $govuk-spacing-scale-8;
+
+    @include mq ($from: desktop) {
+      display: inline-block;
+      width: 33.33%;
+      padding-right: 0;
+      vertical-align: top;
+    }
+  }
+
+  .govuk-header__proposition {
+    @include mq ($from: desktop) {
+      display: inline-block;
+      width: 66.66%;
+    }
+  }
+
+  .govuk-header__menu-button {
+    @include govuk-font-regular-16;
+    position: absolute;
+    top: $govuk-spacing-scale-4;
+    right: 0;
+    margin: 0;
+    padding: 0;
+    border: 0;
+    color: $govuk-header-link;
+    background: none;
+
+    &:hover {
+      text-decoration: underline;
+    }
+
+    &::after {
+      @include govuk-shape-arrow($direction: down, $base: 10px);
+      content: "";
+      display: inline-block;
+      margin-left: $govuk-spacing-scale-1;
+    }
+
+    @include govuk-focusable;
+
+    @include mq ($from: tablet) {
+      top: $govuk-spacing-scale-3;
+    }
+
+    @include mq ($from: desktop) {
+      display: none;
+    }
+  }
+
+  .govuk-header__menu-button--open {
+    &::after {
+      @include govuk-shape-arrow($direction: up, $base: 10px);
+      display: inline-block;
```

Comment:
It's a shame we have to set `display: inline-block` twice to override the mixin. Completely NOT a problem with the PR. I've raised an [issue.](https://github.com/alphagov/govuk-frontend/issues/698). It might worth linking to the issue in the code to say the second `display: inline-block` can be removed if the issue is fixed. Up to you @alex-ju 🙂

Reply:
https://github.com/alphagov/govuk-frontend/pull/701

Reply:
@hannalaakso: @igloosi created a separate PR to deal with this